### PR TITLE
I added a few pages translated into Turkish.

### PR DESCRIPTION
**Pull request by hamer1818**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- I added a few pages translated into Turkish.

### CODE REVIEW

File: docs/src/content/docs/tr/guides/css-and-tailwind.mdx [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+Bu değişkenler, metin ve arka plan renkleri için kullanılan bir dizi gri tonu ve gezinmede bağlantıları ve geçerli öğeleri vurgulamak için kullanılan bir vurgu rengi ile arayüzün genelinde kullanılır.
+
+### Renk tema düzenleyici
+
+Starlight'ın vurgu ve gri renk paletlerini değiştirmek için aşağıdaki kaydırıcıları kullanın.
+Karanlık ve açık önizleme alanları sonuç renklerini gösterecek ve sayfanın tamamı ayrıca değişikliklerinizi önizlemek için güncellenecektir.
+
+Değişikliklerinizden memnun olduğunuzda, aşağıdaki CSS veya Tailwind kodunu kopyalayın ve projenizde kullanın.
+
+import ThemeDesigner from '../../../components/theme-designer.astro';
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/css-and-tailwind.mdx:247]:
```suggestion
import ThemeDesigner from '../../../../components/theme-designer.astro';
```

File: docs/src/content/docs/tr/guides/customization.mdx [hunk 2]
```diff
@@ -0,0 +1,427 @@
+---
+title: Starlight'ı Özelleştirme
+description: Starlight sitenizi logonuz, özel yazı tipleri, giriş sayfası tasarımı ve daha fazlası ile özelleştirmeyi öğrenin.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import FileTree from '../../../components/file-tree.astro';
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/customization.mdx:7]:
```suggestion
import FileTree from '../../../../components/file-tree.astro';
```

File: docs/src/content/docs/tr/guides/i18n.mdx [hunk 3]
```diff
@@ -0,0 +1,221 @@
+---
+title: Uluslararasılaştırma (i18n)
+description: Starlight sitesinin çoklu dil desteğini etkinleştirmek için nasıl yapılandırılacağını öğrenin.
+---
+
+import FileTree from '../../../components/file-tree.astro';
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/guides/i18n.mdx:6]:
```suggestion
import FileTree from '../../../../components/file-tree.astro';
```

File: docs/src/content/docs/tr/reference/configuration.mdx [hunk 4]
```diff
@@ -0,0 +1,457 @@
...
+
+Bu site için varsayılan dilin belirlendiği yerdir.
+Değer, [`locales`](#locales) nesnesinin anahtarlarından biriyle eşleşmelidir.
+(Varsayılan diliniz [root locale](#root-locale) ise bu adımı atlayabilirsiniz.)
+
+Varsayılan yerel dil, çeviriler eksik olduğunda yedek içerik sağlamak için kullanılır.
+
+### `social`
+
+import SocialLinksType from '../../../components/social-links-type.astro';
```
Comment (HiDeoo) [inline docs/src/content/docs/tr/reference/configuration.mdx:311]:
```suggestion
import SocialLinksType from '../../../../components/social-links-type.astro';
```

**Review by HiDeoo — COMMENTED** [review]
Thanks for the translations :raised_hands:

I have not reviewed any of the translations yet but I am suggesting a few changes to make the build pass in case this can help.

To explain what is happening, some relative import paths need to be updated as the Turkish translations are in a nested `tr` folder compared to the English ones.

**Comment by hamer1818** [comment]
> Thanks for the translations :raised_hands:
> 
> I have not reviewed any of the translations yet but I am suggesting a few changes to make the build pass in case this can help.
> 
> To explain what is happening, some relative import paths need to be updated as the Turkish translations are in a nested `tr` folder compared to the English ones.

I never thought of this, thank you.

File: docs/src/content/docs/tr/reference/overrides.md [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+
+### İçerik
+
+Bu bileşenler sayfa içeriğinin ana sütununda görüntülenir.
+
+#### `Banner`
+
+**Varsayılan component:** [`Banner.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/Banner.astro)
+
+Her sayfanın üstünde görüntülenen [`banner`](/reference/frontmatter#banner) bileşeni.
```
Comment (delucis) [inline docs/src/content/docs/tr/reference/overrides.md:308]:
```suggestion
Her sayfanın üstünde görüntülenen [`banner`](/tr/reference/frontmatter#banner) bileşeni. 
```

File: docs/src/content/docs/tr/reference/overrides.md [hunk 6]
```diff
@@ -0,0 +1,375 @@
...
+**Varsayılan component:** [`FallbackContentNotice.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/FallbackContentNotice.astro)
+
+Mevcut dil için bir çeviri mevcut olmayan sayfalarda kullanıcıların gördüğü bildirim. 
+Yalnızca çok dilli sitelerde kullanılır.
+
+#### `Hero`
+
+**Varsayılan component:** [`Hero.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/Hero.astro)
+
+Frontmatter’da [`hero`](/reference/frontmatter#hero) ayarlandığında sayfanın üstünde görüntülenen bir bileşen.
```
Comment (delucis) [inline docs/src/content/docs/tr/reference/overrides.md:336]:
```suggestion
Frontmatter’da [`hero`](/tr/reference/frontmatter#hero) ayarlandığında sayfanın üstünde görüntülenen bir bileşen. 
```

File: docs/src/content/docs/tr/guides/i18n.mdx [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+   	"sidebarNav.accessibleLabel": "Ana",
+   	"tableOfContents.onThisPage": "Bu sayfada",
+   	"tableOfContents.overview": "Genel Bakış",
+   	"i18n.untranslatedContent": "Bu içerik dilinizde henüz mevcut değil.",
+   	"page.editLink": "Sayfayı düzenle",
+   	"page.lastUpdated": "Son Güncelleme:",
+   	"page.previousLink": "Sonraki",
+   	"page.nextLink": "Önceki",
+   	"404.text": "Sayfa bulunamadı. URL yi kontrol edin veya arama çubuğunu deneyin."
+   }
```
Comment (delucis) [inline docs/src/content/docs/tr/guides/i18n.mdx:201]: This should be left in English as indicated by the line immediately above this code block:

> İşte Starlight'ın varsayılan İngilizce değerleri:


```suggestion
   {
   	"skipLink.label": "Skip to content",
   	"search.label": "Search",
   	"search.shortcutLabel": "(Press / to Search)",
   	"search.cancelLabel": "Cancel",
   	"search.devWarning": "Search is only available in production builds. \nTry building and previewing the site to test it out locally.",
   	"themeSelect.accessibleLabel": "Select theme",
   	"themeSelect.dark": "Dark",
   	"themeSelect.light": "Light",
   	"themeSelect.auto": "Auto",
   	"languageSelect.accessibleLabel": "Select language",
   	"menuButton.accessibleLabel": "Menu",
   	"sidebarNav.accessibleLabel": "Main",
   	"tableOfContents.onThisPage": "On this page",
   	"tableOfContents.overview": "Overview",
   	"i18n.untranslatedContent": "This content is not available in your language yet.",
   	"page.editLink": "Edit page",
   	"page.lastUpdated": "Last updated:",
   	"page.previousLink": "Next",
   	"page.nextLink": "Previous",
   	"404.text": "Page not found. Check the URL or try using the search bar."
   }
```

File: docs/src/content/docs/tr/guides/i18n.mdx [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+	"pagefind.filters_label": "Filtreler",
+	"pagefind.zero_results": "[ARAMA_TERIMI] için sonuç bulunamadı",
+	"pagefind.many_results": "[ARAMA_TERIMI] için [SAYI] sonuç",
+	"pagefind.one_result": "[ARAMA_TERIMI] için [SAYI] sonuç",
+	"pagefind.alt_search": "[ARAMA_TERIMI] için sonuç bulunamadı. Bunun yerine [FARKLI_TERIM] sonuçları gösteriliyor",
+	"pagefind.search_suggestion": "[ARAMA_TERIMI] için sonuç bulunamadı. Aşağıdaki aramalardan birini deneyin:",
+	"pagefind.searching": "[ARAMA_TERIMI] aranıyor..."
+	}
+
+   ```
```
Comment (delucis) [inline docs/src/content/docs/tr/guides/i18n.mdx:221]: Same here:

````suggestion
   ```json
   {
   	"pagefind.clear_search": "Clear",
   	"pagefind.load_more": "Load more results",
   	"pagefind.search_label": "Search this site",
   	"pagefind.filters_label": "Filters",
   	"pagefind.zero_results": "No results for [SEARCH_TERM]",
   	"pagefind.many_results": "[COUNT] results for [SEARCH_TERM]",
   	"pagefind.one_result": "[COUNT] result for [SEARCH_TERM]",
   	"pagefind.alt_search": "No results for [SEARCH_TERM]. Showing results for [DIFFERENT_TERM] instead",
   	"pagefind.search_suggestion": "No results for [SEARCH_TERM]. Try one of the following searches:",
   	"pagefind.searching": "Searching for [SEARCH_TERM]..."
   }
   ```
````

File: docs/src/content/docs/tr/guides/overriding-components.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+2. Starlight bileşenini değiştirmek için kullanılacak bir Astro bileşeni oluşturun.
+   Bu örnek bir iletişim bağlantısı oluşturur. 
+
+   ```astro
+  ---
+  // src/components/EmailLink.astro
+  import type { Props } from '@astrojs/starlight/props';
+  ---
+
+  <a href="mailto:houston@example.com">Bana E-posta Gönder</a>
```
Comment (delucis) [inline docs/src/content/docs/tr/guides/overriding-components.md:36]: Looks like the indentation here is causing issues:

image


This should fix it:

```suggestion
   ---
   // src/components/EmailLink.astro
   import type { Props } from '@astrojs/starlight/props';
   ---

   <a href="mailto:houston@example.com">Bana E-posta Gönder</a>
```

File: docs/src/content/docs/tr/guides/overriding-components.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+    integrations: [
+      starlight({
+        title: 'Geçersiz Kılmalarla Benim Belgelerim',
+        components: {
+          // Varsayılan `SocialIcons` bileşenini geçersiz kıl.
+          SocialIcons: './src/components/EmailLink.astro',
+        },
+      }),
+    ],
+  });
```
Comment (delucis) [inline docs/src/content/docs/tr/guides/overriding-components.md:56]: Same is likely the case here although we can’t see it yet:
```suggestion
   // astro.config.mjs
   import { defineConfig } from 'astro/config';
   import starlight from '@astrojs/starlight';

   export default defineConfig({
     integrations: [
       starlight({
         title: 'Geçersiz Kılmalarla Benim Belgelerim',
         components: {
           // Varsayılan `SocialIcons` bileşenini geçersiz kıl.
           SocialIcons: './src/components/EmailLink.astro',
         },
       }),
     ],
   });
```

File: docs/src/content/docs/tr/reference/configuration.mdx [hunk 11]
```diff
@@ -0,0 +1,457 @@
...
+        rel: 'ikon',
+        href:'/images/favicon.ico',
+        boyutlar: '32x32',
+      },
+    },
+  ],
+});
+```
+
+### `titleDelimiter | başlıkAyırıcı`
```
Comment (delucis) [inline docs/src/content/docs/tr/reference/configuration.mdx:432]: In general we don’t translate these headings which are code options, so I guess this should match that?

```suggestion
### `titleDelimiter`
```

File: docs/src/content/docs/tr/reference/frontmatter.md [hunk 12]
```diff
@@ -0,0 +1,353 @@
...
+description: Üzerinde çalıştığım projeyi daha iyi anlayın.
+---
+
+Hakkında sayfasına, Hoş geldiniz!
+
+```
+
+## Frontmatter alanları
+
+### `title` (Gerekli)
```
Comment (delucis) [inline docs/src/content/docs/tr/reference/frontmatter.md:20]: Noticed that “gerekli” is lowercase in the configuration reference. Should it be the same here?

```suggestion
### `title` (gerekli)
```

File: docs/src/content/docs/tr/reference/frontmatter.md [hunk 13]
```diff
@@ -0,0 +1,353 @@
...
+
+**Tür:** `string`
+
+Sayfa açıklaması, sayfa meta verileri için kullanılır ve arama motorları tarafından ve sosyal medya önizlemelerinde kullanılır.
+
+### `editUrl`
+
+**Tür:** `string | boolean`
+
+[global `editLink` yapılandırmasını](/reference/configuration/#editlink) geçersiz kılar. Belirli bir sayfa için "Sayfayı Düzenle" bağlantısını devre dışı bırakmak için `false` olarak ayarlayın veya bu sayfanın içeriğinin düzenlenebileceği alternatif bir URL sağlayın.
```
Comment (delucis) [inline docs/src/content/docs/tr/reference/frontmatter.md:36]: Should the start of the sentence here be capitalised?

```suggestion
[Global `editLink` yapılandırmasını](/tr/reference/configuration/#editlink) geçersiz kılar. Belirli bir sayfa için "Sayfayı Düzenle" bağlantısını devre dışı bırakmak için `false` olarak ayarlayın veya bu sayfanın içeriğinin düzenlenebileceği alternatif bir URL sağlayın.
```

File: docs/src/content/docs/tr/reference/frontmatter.md [hunk 14]
```diff
@@ -0,0 +1,353 @@
...
+
+### `head`
+
+**Tür:** [`HeadConfig[]`](/reference/configuration/#headconfig)
```
Comment (delucis) [inline docs/src/content/docs/tr/reference/frontmatter.md:40]:
```suggestion
**Tür:** [`HeadConfig[]`](/tr/reference/configuration/#headconfig)
```

File: docs/src/content/docs/tr/reference/frontmatter.md [hunk 15]
```diff
@@ -0,0 +1,353 @@
...
+---
+title: İçerik tablosu olmayan sayfa
+tableOfContents: false
+---
+```
+
+### `template`
+
+**Tür:** `'doc' | 'splash'`  
+**varsayılan:** `'doc'`
```
Comment (delucis) [inline docs/src/content/docs/tr/reference/frontmatter.md:79]: Noticing here that “Tür” capitalised but “varsayılan” is not. Should they be consistent?

There are a few places on this page like this.

File: docs/src/content/docs/tr/reference/overrides.md [hunk 16]
```diff
@@ -0,0 +1,375 @@
+---
+title: Geçersiz Kılma Başvurusu
+description: Starlight tarafından desteklenen geçersiz kılma işlemleri için desteklenen bileşenlerin ve bileşen özelliklerinin genel bir incelemesi.
+tableOfContents:
+  maxHeadingLevel: 4
+---
+
+Starlight'ın yerleşik bileşenlerini, Starlight'ın [`components`](/reference/configuration#components) yapılandırma seçeneğinde yer değiştirme bileşenlerinin yolunu sağlayarak geçersiz kılabilirsiniz.
```
Comment (delucis) [inline docs/src/content/docs/tr/reference/overrides.md:8]:
```suggestion
Starlight'ın yerleşik bileşenlerini, Starlight'ın [`components`](/tr/reference/configuration#components) yapılandırma seçeneğinde yer değiştirme bileşenlerinin yolunu sağlayarak geçersiz kılabilirsiniz.
```

**Review by delucis — CHANGES_REQUESTED** [review]
Thank you for this big PR @hamer1818! I appreciate you taking the time to help make our docs available to readers of Turkish.

Took me a while to go through, but left quite a few comments. 90% of these are making sure that links point to the `/tr/` version of pages and use the correct `#` links for headings translated to Turkish. I also noticed a few formatting fixes and left a couple of questions.

**Comment by hamer1818** [comment]
i updated a new one

**Pull request closed without merging**